app/src/main/java/it/niedermann/owncloud/notes/android/activity/NotesListViewActivity.java

```diff
@@ -233,6 +220,27 @@ protected void onCreate(Bundle savedInstanceState) {
 
     @Override
     protected void onResume() {
+        // TODO This needs to skip if there are no detected changes between localAccount id
+        // and current active single sign on account
+        try {
+            selectAccount(SingleAccountHelper.getCurrentSingleSignOnAccount(getApplicationContext()).name);
+        } catch (NoCurrentAccountSelectedException | NextcloudFilesAppAccountNotFoundException e) {
+            handleNotAuthorizedAccount();
```

Comment:
@dan0xii this breaks the migration of an account from a previous version:

`handleNotAuthorizedAccount()` pops up a dialog to import/authorize this account, after pressing `cancel` or `ok` the activity is resumed which leads to some kind of endless-loop.

Reply:
Maybe we should call this in `onResume()` **and** in `onCreate()` - and call `handleNotAuthorizedAccount()` only in `onCreate()` while removing the exception handling in `onResume()`...?

Or a boolean flag where we remember if we have asked the user already to import/authorize the account?

Reply:
I went for a boolean flag for now to simply fix this.